nwb_web_gui/dashapps/pages/converter/example_schemas/schema_metadata_all.json

```diff
@@ -242,23 +254,92 @@
             },
             "optical_channels": {
               "type": "array",
-              "description": "One of possibly many groups storing channel-specific data.",
+              "description": "One of possibly many groups storing channel-specific data",
               "items": [
                 {"$ref":  "#/definitions/OpticalChannel"},
                 {"$ref":  "#/definitions/OpticalChannel"}
               ]
             }
           }
+        },
+        "ImageSegmentation1": {
+          "tag": "pynwb.ophys.ImageSegmentation",
+          "title": "ImageSegmentation",
+          "type": "object",
+          "required": ["name", "plane_segmentations"],
+          "properties": {
+            "name":  {
+              "type": "string",
+              "description": "the name of this container"
+            },
+            "plane_segmentations": {
+              "type": "array",
+              "description": "PlaneSegmentation to store in this interface",
+              "items": [
+                {"$ref":  "#/definitions/PlaneSegmentation"},
+                {"$ref":  "#/definitions/PlaneSegmentation"}
```

Comment:
I don't think you need two of them here

Reply:
they're there merely as examples of lists of items inside `pynwb.core.MultiContainerInterface` containers such as `pynwb.ophys.ImageSegmentation` and `pynwb.behavior.Position`.
We can safely remove them, I think it could be interesting for illustration

Reply:
Fine to illustrate with metadata files, but doing so in a json schema is incorrect. There is an example hey I'm json schema that you can use for this if you want